docs/CHANGELOG.md

```
@@ -3,6 +3,23 @@
+javac.  The Checker Framework can type-check any Java project, including projects
+that compile to Java 8 or 11 bytecodes and run on JRE versions 8 or 11.
+
+The type qualifiers and utility libraries in `checker-qual.jar` and
+`checker-util.jar` still use Java 11 bytecode.  Thus, they may be used in
+projects that run under Java 11 or later.
+
+### Implementation details
+
+### Closed issues
+
+## Version 3.54.1 (2026-04-01)
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)